jekyll-export-cli.php

```diff
@@ -16,12 +16,54 @@
  *
  * Must be run in the wordpress-to-jekyll-exporter/ directory.
  */
-require '../../../wp-load.php';
+
+// Uncomment for extra replace options
+
+$wordpress_path = "../../../";
+//$wordpress_path = "/home/wordpress/";
+
+require $wordpress_path . 'wp-load.php';
 require_once 'jekyll-exporter.php'; // Ensure plugin is "activated".
 
+$config["site_url"] = "https://www.mywebsite.fr/";
+
+$config["extra_remove_site_url"] = $config["site_url"];  // Will remove site url to get relative URLs
+$config['extra_code'] = true; // Will retrieve lang and protect pre/code
+$config['extra_img_style'] = true; // Will retrieve style from image
+$config['extra_tag_spoiler'] = true; // Will convert spoiler tags
```

Comment:
Can the config options be set as CLI flags to prevent needing to modify the source code to customize the behavior?

Reply:
I am not really sure if this will be handy to use: in my migration process I had to run several times the exporter as I was seeing things in the output that were not right. Giving options in command line would have been very painful in my case. To start with, I have added the ability to define a local config outside the code that will be included if present.

If you still want this as command line parameters, what would be the command line framework to use?